v1/metrics.yaml

```diff
@@ -429,115 +439,860 @@ paths:
                 x-client-ip: '{{x-client-ip}}'
       x-monitor: false
 
-definitions:
-  listing:
-    description: The result format for listings
-    required:
-      - items
-    properties:
-      items:
-        type: array
-        items:
-          type: string
+  ########################################
+  # Edited Pages
+  ########################################
+  /edited-pages/:
+    get:
+      tags:
+        - Analytics Edited-Pages Metrics API
+      summary: List mediawiki edited-pages metrics related API entry points.
+      description: |
+        This is the root of mediawiki edited-pages metrics data endpoints. If browsing the interactive documentation, see the specifics for each endpoint below.
 
-  pageview-article:
-    properties:
-      items:
-        type: array
-        items:
-          properties:
-            project:
-              type : string
-            access:
-              type : string
-            article:
-              type: string
-            agent:
-              type: string
-            granularity:
-              type: string
-            timestamp:
-              # the hourly timestamp will be stored as YYYYMMDDHH
-              type: string
-            views:
-              type: integer
-              format: int64
+        Stability: [unstable](https://www.mediawiki.org/wiki/API_versioning#Stable)
```

Comment:
the tag says 'unstable' but the link goes to 'stable'

Reply:
Done !